tests/integ/modin/io/test_read_snowflake_query_call.py

```diff
@@ -46,5 +46,5 @@ def filter_by_role(session, table_name, role):
 @sql_count_checker(query_count=5, sproc_count=2)
 def test_read_snowflake_call_system_function(session):
     df = pd.read_snowflake("CALL SYSTEM$TYPEOF(TRUE)")
-    native_df = native_pd.DataFrame(session.sql("CALL SYSTEM$TYPEOF(TRUE)").collect())
+    native_df = session.sql("CALL SYSTEM$TYPEOF(TRUE)").collect().to_pandas()
```

Comment:
It seems `collect()` should be removed here.

Reply:
good catch, thank you!